frontend/src/MakerApp.tsx

```diff
@@ -149,24 +149,24 @@ export default function App() {
 
             <Tabs marginTop={5}>
                 <TabList>
-                    <Tab>Running [{running.length}]</Tab>
                     <Tab>Open [{open.length}]</Tab>
+                    <Tab>Accept / Reject [{acceptOrReject.length}]</Tab>
```

Comment:
I was first a bit confused and thought this would be a `Accepted`/`Rejected` category. I can't come up with a better term though.

Reply:
The maker has to make a decision there, so it should be easy for the maker to see those. That's why I opted for it. But yeah, it is a bit weird. Would change this once we have used it and find it bad.